Resources/Prototypes/_Scp/Recipes/Crafting/bots.yml

```
@@ -0,0 +1,12 @@
+- type: construction
+  name: roomba
+  id: roomba
+  graph: RoombaConstruct
+  startNode: start
+  targetNode: bot
+  category: construction-category-utilities
+  objectType: Item
+  description: This bot wanders around the station, mopping up any puddles it sees.
+  icon:
+    sprite: _Scp/Mobs/Silicon/Bots/roomba.rsi
+    state: roomba
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)